src/index.ts

```diff
@@ -120,9 +145,61 @@ const GlobalIdExtensionsTweakFieldsPlugin: Plugin = function(builder) {
       });
     }, fields);
   });
+
+  // add deprecations
+  builder.hook("GraphQLObjectType:fields:field", (field, { inflection }, context) => {
```

Comment:
We've found that not destructuring in the args makes it easier for other people to contribute to the plugin.

```suggestion
  builder.hook("GraphQLObjectType:fields:field", (field, build, context) => {
    const { inflection } = build;
```